joplin/settings.py

```diff
@@ -139,6 +148,9 @@
 TIME_ZONE = 'UTC'
 USE_TZ = True
 
+MODELTRANSLATION_DEFAULT_LANGUAGE = 'en'
```

Comment:
🔤

Reply:
In my local environment, the migration process inserted the data without errors; however, when inserting a multi-language entry it kept defaulting to Arabic; for example, when it inserted a page, the "title_en", "title_e"s, "title_ar", and "title_vi" columns inserted the content fine, but the "title" column defaulted to title_ar.

I tried manually assigning the values title = title_en, but even that would not work it would re-write the entire title_ar column with it. When inserted this line, the problem went away.

Reply:
yeah, I remember you explaining that. Fixed in just one line 👍